addons/editor-dark-mode/experimental_editor.css

```diff
@@ -375,6 +378,7 @@ img[class*="tool-select-base_tool-select-icon_"],
 [class*="tag-button_tag-button_"]:not([class*="tag-button_active_"]),
 [class*="action-menu_button_"],
 [class*="context-menu_menu-item_"]:not([class*="context-menu_menu-item-danger_"]):hover,
+[class*="webgl-modal_button-row"] > button,
```

Comment:
The border color also needs to be set. It currently looks like this with the TurboWarp Dark preset:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/7fef9bb3-f0bc-457c-9004-4c682c1851fb)